Examples/Java/Sources/Board.java

```diff
@@ -21,42 +20,289 @@
 import java.util.Date;
 import java.util.List;
 import java.util.Map;
+import java.util.Objects;
 import java.util.Set;
 
-@AutoValue
-public abstract class Board {
+public class Board {
 
-
-    public abstract @SerializedName("id") @Nullable String identifier();
-    public abstract @SerializedName("contributors") @Nullable Set<User> contributors();
-    public abstract @SerializedName("counts") @Nullable Map<String, Integer> counts();
-    public abstract @SerializedName("created_at") @Nullable Date createdAt();
-    public abstract @SerializedName("creator") @Nullable Map<String, String> creator();
-    public abstract @SerializedName("description") @Nullable String descriptionText();
-    public abstract @SerializedName("image") @NonNull Image image();
-    public abstract @SerializedName("name") @Nullable String name();
-    public abstract @SerializedName("url") @Nullable String url();
-    public static Builder builder() {
-        return new AutoValue_Board.Builder();
-    }
-    abstract Builder toBuilder();
-    public static TypeAdapter<Board> jsonAdapter(Gson gson) {
-        return new AutoValue_Board.GsonTypeAdapter(gson);
-    }
-    @AutoValue.Builder
-    public abstract static class Builder {
-    
+    @SerializedName("id") private @Nullable String identifier;
+    @SerializedName("contributors") private @Nullable Set<User> contributors;
+    @SerializedName("counts") private @Nullable Map<String, Integer> counts;
+    @SerializedName("created_at") private @Nullable Date createdAt;
+    @SerializedName("creator") private @Nullable Map<String, String> creator;
+    @SerializedName("description") private @Nullable String descriptionText;
+    @SerializedName("image") private @NonNull Image image;
+    @SerializedName("name") private @Nullable String name;
+    @SerializedName("url") private @Nullable String url;
```

Comment:
Not important now but some newlines would likely clean this up a bit

Reply:
Done.